tests/test_gpu_examples.py

```diff
@@ -1671,6 +1671,117 @@ def forward(self, x):
         assert not torch.allclose(output_finetuned_pissa, output_converted, atol=tol, rtol=tol)
 
 
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires a GPU")
+class TestOLoRA:
+    r"""
+    Tests for OLoRA to ensure that it reduces the quantization error compared to normal LoRA quantization.
+    """
+
+    # The error factor indicates by how much the quantization error should be decreased when using OLoRA compared to
+    # quantization without OLoRA. Thus 1.03 means that the error should be decreased by 3% at least. This is a very
+    # conservative value to prevent flakiness, in practice most gains are > 1.5
+    error_factor = 1.03
```

Comment:
I checked the errors for all the tests and found that 1.03 is very conservative. Something like 1.3 would also work, i.e. a 10x increase in margin (though not 1.5 as the comment suggests, which probably was just copied from PiSSA?). What did you observe on your machine?

We should probably not go too high here to prevent flakiness, but if you can confirm my observations, how about going to 1.2?